Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -10,15 +10,33 @@
   }
 
   &--dark {
-    @apply bg-black;
+    background: url(../../img/background-pattern-dark.svg);
 
     .section__content {
       .section__content__p {
+        @apply block text-white;
+      }
+
+      .section__content__title {
+        @apply text-orange normal-case;
+      }
+
+      .item-grid .grid-content .grid-item .item-details {
         @apply text-white;
       }
+    }
+  }
+
+  &--light {
+    background: url(../../img/background-pattern-light.svg);
+
+    .section__content {
+      .section__content__p {
+        @apply block text-black;
```

Comment:
Why `block` was added to a CSS class intended to be set on a `<p>` tag which is a block-type element by default?